sdk/translation/ai-translation-document-rest/src/documentTranslationClient.ts

```diff
@@ -24,7 +24,7 @@ export default function createClient(
   { apiVersion = "2024-05-01", ...options }: DocumentTranslationClientOptions = {},
 ): DocumentTranslationClient {
   const endpointUrl = options.endpoint ?? options.baseUrl ?? `${endpointParam}/translator`;
-  const userAgentInfo = `azsdk-js-ai-translation-document-rest/1.0.0-beta.1`;
+  const userAgentInfo = `azsdk-js-ai-translation-document-rest/1.0.0`;
```

Comment:
oh so we release 1.0.0 with the wrong telemetry info?!  

nitpick: may be worth to add an entry for this in package.json to have bot auto-increment version after a release.

Reply:
Addressed in https://github.com/Azure/azure-sdk-for-js/pull/33640/commits/18ae14fd1dd8b4ebdd54c26850cf62a9b3db536a